sdk/grpc-sink/src/main/java/com/google/cloud/trace/grpc/v1/GrpcTraceSink.java

```diff
@@ -50,13 +50,15 @@ public GrpcTraceSink(String apiHost, Credentials credentials) {
   }
 
   @Override
-  public void receive(Trace trace) {
-    Traces.Builder traceBuilder = Traces.newBuilder().addTraces(trace);
-
+  public void receive(Traces traces) {
+    if (traces.getTracesCount() == 0) {
+      return;
+    }
+    String projectId = traces.getTraces(0).getProjectId();
```

Comment:
This makes me think two things:
The TraceSink interface should take a project id.
The API Trace should not have a project id.